### PR TITLE
KOGITO-6607 adding user jdoe to requests

### PR DESCRIPTION
**Pull request by uteegozi**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `main` branch!**

- [x ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ x] Pull Request contains link to the JIRA issue
- [ x] Pull Request contains link to any dependent or related Pull Request
- [ x] Pull Request contains description of the issue
- [ x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
After assigning user tasks to a specific user in PR https://github.com/kiegroup/kogito-examples/pull/1179 requests need to be sent with the relevant user as parameter. 
The user jdoe was added to the example requests on the Readme file of the Kogito Extended Travel example.

Jira Issue: https://issues.redhat.com/browse/KOGITO-6607
Related PR: https://github.com/kiegroup/kogito-examples/pull/1179
</summary>

</details>

### CODE REVIEW

**Review by cristianonicolai — APPROVED** [review]

**Comment by cristianonicolai** [comment]
ok to test

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: kogito-quarkus-examples/kogito-travel-agency/extended/travels/README.md [hunk 1]
```diff
@@ -328,28 +328,28 @@ Cancels travel request with given id
 curl -X DELETE http://localhost:8080/travels/{uuid}
 ```
 
-### GET /travels/{id}/tasks
+### GET /travels/{id}/tasks?user=jdoe
```
Comment (radtriste) [inline kogito-quarkus-examples/kogito-travel-agency/extended/travels/README.md]: Please remove this from titles, I don't think it makes sense. Only the command is useful.

**Review by radtriste — CHANGES_REQUESTED** [review]
Could you also add the `visas` changes ?

File: kogito-quarkus-examples/kogito-travel-agency/extended/travels/README.md (hunk 1 above)
Comment (uteegozi) [reply kogito-quarkus-examples/kogito-travel-agency/extended/travels/README.md]: Will remove from titles and also update Visas Readme

Comment (uteegozi) [reply kogito-quarkus-examples/kogito-travel-agency/extended/travels/README.md]: [b73e1de](https://github.com/kiegroup/kogito-examples/pull/1237/commits/b73e1de76033d3b82d248a93055a157949aa3b61)
In addition removing user jdoe from the titles we also added user alice to the relevant visa requests and fixed payloads where needed.

**Review by radtriste — APPROVED** [review]
tested all good for me. thx !

**Pull request merged**